cdk_infra/lib/app.ts

```diff
@@ -19,4 +20,7 @@ new MSKClustersStack(app, 'msk-clusters', {
   env: envDefault,
   eksVPC: vpcStack.vpc
 });
+new ConfigMapProvidersStack(app, 'config-map-providers', {
+  env: envDefault
```

Comment:
Should we be specifying a region here to deploy the cfn template to? Would an error arise if someone tried to deploy this CFN template in multiple regions?

Reply:
your question is applicable to all the other resources as well. 

We are relying on region definition from here: https://github.com/aws-observability/aws-otel-test-framework/blob/terraform/.github/workflows/cdk-deploy.yml#L26

Reply:
I don't think this is true. EKS and MSK are not globally unique in the way that s3 bucket names are.

Reply:
Perhaps there are some resources inside those stacks that are globally unique, IAM Roles(?), and that would be a fair critique.

Reply:
I see, you are referring to the globally unique names. I think we can add the region name in the bucket name as well.